packages/amplify-graphql-api-construct-tests/src/utils/auth.ts

```diff
@@ -0,0 +1,116 @@
+import {
```

Comment:
Thanks for adding these utilities. I've noticed that these were missing as well on Friday.